eslint-plugin/src/index.ts

```diff
@@ -0,0 +1,56 @@
+
+import { TSDocParser } from "@microsoft/tsdoc";
+import { allTsdocMessageIds } from "@microsoft/tsdoc/lib/parser/TSDocMessageId";
+import * as eslint from "eslint";
+import * as ESTree from "estree";
+
+const messageIds: {[x: string]: string} = {};
+
+allTsdocMessageIds.forEach((messageId: string) => {
+    messageIds[messageId] = messageId + ": {{ unformattedText }}";
+});
+
+module.exports = {
+    rules: {
+        "tsdoc-comments": {
+            meta: {
+                messages: messageIds,
+                type: "problem",
+                docs: {
+                    description: "Validates tsdoc comments",
+                    category: "Typescript",
+                    recommended: false,
+                    url: "https://github.com/microsoft/tsdoc"
+                }
+            },
+            create: function(context: eslint.Rule.RuleContext) {
```

Comment:
Use a lambda function here.

```suggestion
            create: (context: eslint.Rule.RuleContext) => {
```